package.json

```diff
@@ -115,6 +116,7 @@
     "eslint-plugin-react-hooks": "^4.3.0",
     "eslint-plugin-tsdoc": "^0.2.4",
     "find-yarn-workspace-root": "^1.2.1",
+    "follow-redirects": "^1.14.7",
```

Comment:
can probably remove follow-redirects and @types/follow-redirects, they were removed from the text-index cli command

Reply:
👍 removed.